server/core/src/https/views/login.rs

```diff
@@ -29,14 +29,14 @@ use crate::https::{
 };
 
 use std::str::FromStr;
-
+use time::OffsetDateTime;
 use super::{HtmlTemplate, UnrecoverableErrorView};
 
 #[derive(Template)]
 #[template(path = "login.html")]
-struct LoginView<'a> {
-    username: &'a str,
-    remember_me: bool,
+pub struct LoginView<'a> {
+    pub(crate) username: &'a str,
```

Comment:
Why did you change the visibility of these?

Reply:
This was so the _to_be_replaced_ get_transformer function could construct LoginView by itself.

Reply:
I think we drop this part and if there is a specific issue we follow up with it in isolation (especially because I just did a ton of changes to the login flow)

Reply:
And because you shouldn't need to construct the LoginView anywhere else. You should redirect to the login page, not render it inplace.